libr/Makefile

```diff
@@ -47,6 +51,9 @@ atombios-install:
 	cp -f anal/p/anal_atombios.$(LIBEXT) $(R2PM_PLUGDIR)
 	cp -f bin/p/bin_atombios.$(LIBEXT) $(R2PM_PLUGDIR)
 
+ba2-install:
```

Comment:
just copypaste the lines above from another asm/anal plugin

Reply:
Updated